src/ui/tests/orders/criticalPath/replaceDeleteManager.spec.ts

```diff
@@ -0,0 +1,103 @@
+        1,
+        token,
+      );
+      orderId = result._id;
+
+      managerUsername = MOCK_MANAGER_NASTYA.username;
+      managerFirstName = MOCK_MANAGER_NASTYA.firstName;
+      managerLastName = MOCK_MANAGER_NASTYA.lastName;
+
+      await homeUIService.openAsLoggedInUser();
+      await homeUIService.openModule('Orders');
+
+      await ordersPage.clickDetailsButton(orderId);
+      await orderDetailsPage.waitForOpened();
+    },
+  );
+
+  test(
+    'Replace assigned manager',
+    { tag: [TAGS.ORDERS] },
+    async ({ orderDetailsPage, ordersPage, notificationsModal }) => {
+      //открываем модалку изменения менеджера
+      await orderDetailsPage.topPanel.clickEditAssignedManagerButton();
+      await orderDetailsPage.waitForOpened();
+
+      //выбираем отличного от назначнного
+      await orderDetailsPage.editAssignedManagerInOrderModal.clickManagerListItem(
+        managerUsername,
+      );
+      await orderDetailsPage.editAssignedManagerInOrderModal.clickSaveButton();
+      await orderDetailsPage.waitForSpinner();
+
+      //проверка всплывающего уведомления о назначении менелджера
+      await orderDetailsPage.waitForNotification(
+        UI_TEXTS.NOTIFICATION_TOASTER.MANAGER_SUCCESSFULLY_ASSIGNED,
+      );
+      await orderDetailsPage.waitForSpinner();
+
+      //проверка assigned manager в поле заказа
+      const updatedAssignedManager =
+        await orderDetailsPage.topPanel.getAssignedManagerName();
+      await expect
+        .soft(updatedAssignedManager, 'Manager name is different')
+        .toBe(managerFirstName + ' ' + managerLastName);
+
+      //проверка уведомления в модалке уведомлений о назначении менеджером
+      await ordersPage.clickOpenNotifications();
+      const notificationText = await notificationsModal.getNotificationText(0);
+      await expect(
+        notificationText,
+        'Should ger notification about assigned',
+      ).toBe(UI_TEXTS.NOTIFICATION.MANAGER_ASSIGNED);
+    },
+  );
+
+  test(
+    'Delete assigned manager',
+    { tag: [TAGS.ORDERS] },
+    async ({ orderDetailsPage, confirmationModal }) => {
+      //удаляем уже назначенного менеджера и ждем открытия модалки
+      await orderDetailsPage.topPanel.clickRemoveAssignedManagerButton();
+      await orderDetailsPage.waitForOpened();
+
+      // подтверждаем удаление
+      await confirmationModal.clickConfirmButton();
+      await orderDetailsPage.waitForSpinner();
+
+      //проверка всплывающего уведомления об удаление менелджера
+      await orderDetailsPage.waitForNotification(
+        UI_TEXTS.NOTIFICATION_TOASTER.MANAGER_SUCCESSFULLY_UNASSIGNED,
+      );
+      await orderDetailsPage.waitForSpinner();
+
+      //проверка что менеджер пропал из поля назначенный менеджер в заказе
+      const assignedManagerLink = orderDetailsPage.topPanel.assignedManagerLink;
+      await expect(
+        assignedManagerLink,
+        'Manager should be empty',
+      ).not.toBeVisible();
```

Comment:
Возможно стоит отталкиваться от ожидаемого поведения юай в проверках
Когда мы удаляем менеджера начинает отображаться определенная кнопка, вот и проверка
```suggestion
      await expect(
        orderDetailsPage.topPanelюassignManagerButton,
        '"Click to select manager" button is not displayed',
      ).toBeVisible();
```